src/Unity.Mathematics/math.cs

```diff
@@ -6647,15 +6782,6 @@ internal static double4 movehl(double4 a, double4 b)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal static uint4 fold_to_uint(double4 x) { return uint4(fold_to_uint(x.x), fold_to_uint(x.y), fold_to_uint(x.z), fold_to_uint(x.w)); }
 
-        [StructLayout(LayoutKind.Explicit)]
-        internal struct IntFloatUnion
-        {
-            [FieldOffset(0)]
-            public int intValue;
-            [FieldOffset(0)]
-            public float floatValue;
-        }
-
         [StructLayout(LayoutKind.Explicit)]
         internal struct LongDoubleUnion
```

Comment:
Can this here also be deleted now?

Reply:
I might be able to but I chose not to because it's used for `lzcnt` and `tzcnt` and I wanted to make the trivial replacements first.